app/views/api/v2/usergroups/main.json.rabl

```diff
@@ -2,4 +2,4 @@ object @usergroup
 
 extends "api/v2/usergroups/base"
```

Comment:
users table has a different `:admin` field. Anyway, I'm not sure that the field is relevant to all views.